src/content/docs/pl/concepts/why-astro.mdx

```diff
@@ -0,0 +1,80 @@
+
+To podejście stoi w kontrze do innych współczesnych JavaScriptowych frameworków, takich jak Next.js, SvetleKit, Nuxt, Remix i podobnych. Wymagają one renderowania na kliencie całej strony i używają serwerowego renderowania głównie, by zaadresować problemy z wydajnością. Takie podejście zostało nazwane **Single Page App (SPA)**, w przeciwieństwie do podejścia **Multi Page App (MPA)**, które stosuje Astro.
+
+Model SPA ma swoje korzyści. Jednak są one okupione dodatkową złożonością i kompromisami związanymi z wydajnością. Te kompromisy obniżają wydajność strony — włączając w to istotne metryki, jak [Czas do Interaktywności (ang. *Time to Interactive* — TTI)](https://web.dev/interactive/) — i nie mają one większego sensu dla stron opartych na treści, gdzie szybkość pierwszego ładowania jest kluczowa.
+
+📚 [Dowiedz się więcej o tym, co czyni architekturę MPA Astro wyjątkową.](/pl/concepts/mpa-vs-spa/)
+
+
+## Domyślnie szybki
+
+Dobra wydajność zawsze jest ważna, lecz dla stron opartych o treści jest *wyjątkowo* krytyczna. To, że niska wydajność negatywnie wpływa na zaangażowanie, konwersje i traci pieniądze, zostało dobrze sprawdzone. Na przykład:
+
+- Każde 100ms szybciej → 1% więcej konwersji ([Mobify](https://web.dev/why-speed-matters/), dodatkowe $380,000/rok)
+- 50% szybciej → 12% więcej sprzedaży ([AutoAnything](https://www.digitalcommerce360.com/2010/08/19/web-accelerator-revs-conversion-and-sales-autoanything/))
+- 20% szybciej → 10% więcej konwersji ([Furniture Village](https://www.thinkwithgoogle.com/intl/en-gb/marketing-strategies/app-and-mobile/furniture-village-and-greenlight-slash-page-load-times-boosting-user-experience/))
+- 40% szybciej → 15% więcej rejestracji ([Pinterest](https://medium.com/pinterest-engineering/driving-user-growth-with-performance-improvements-cfc50dafadd7))
+- 850ms szybciej → 7% więcej konwersji ([COOK](https://web.dev/why-speed-matters/))
+- Każda 1s wolniej → 10% mniej użytkowników ([BBC](https://www.creativebloq.com/features/how-the-bbc-builds-websites-that-scale))
+
+W wielu webowych frameworkach łatwo jest zbudować stronę, która wygląda świetnie podczas developmentu, tylko po to, żeby po zahostowaniu ładowała się boleśnie wolno. Winowajcą często jest JavaScript, ponieważ telefony i mniej wydajne urządzenia użytkowników rzadko dorównują wydajnością laptopom deweloperów.
+
+Magia Astro bierze się z połączenia dwóch wartości opisanych powyżej — skupieniu na treści oraz bazującej na serwerze architekturze MPA — pozwala to podejmować decyzje i dostarczać funkcjonalności, których nie są w stanie dostarczyć inne frameworki. Rezultatem jest niesamowita wydajność dla każdej strony, prosto z pudełka. Naszym celem jest: **Zbudowanie wolnej strony przy użyciu Astro powinno być prawie niemożliwe.**
```

Comment:
```suggestion
Magia Astro bierze się z połączenia dwóch wartości opisanych powyżej — skupieniu na treści oraz bazującej na serwerze architekturze MPA — pozwala to podejmować decyzje i dostarczać funkcjonalności, których nie są w stanie dostarczyć inne frameworki. Rezultatem jest niesamowita wydajność dla każdej strony. Naszym celem jest: **Zbudowanie wolnej strony przy użyciu Astro powinno być prawie niemożliwe.**
```

To "prosto z pudełka" wyglądało tutaj wręcz okropnie, jakby zostało wyrwane prosto z angielskiej wersji i wrzucone do translatora google